src/support/stat.c

```diff
@@ -36,6 +36,7 @@ static const char *const __stats_dsrc_desc[] = {
   "btree: btree compact pages skipped",
   "btree: btree expected number of compact bytes rewritten",
   "btree: btree expected number of compact pages rewritten",
+  "btree: btree number of pages caused to be reconciled during checkpoint",
```

Comment:
number of pages reconciled during checkpoint?

Reply:
Fixed in b70643c7d2d426162a367148b00a48d075a075ed